lib/puma_patch.rb

```diff
@@ -0,0 +1,11 @@
+
+class PumaMapperMiddleware
+  def initialize(app)
```

Comment:
Is this middleware needed for anything right now?

Reply:
Yes, it is the only way for Puma to enter the route and deal with it.

Reply:
Sorry, I don't get it. The only place this middleware is being used is in here https://github.com/theforeman/smart-proxy/pull/613/files#diff-b8fbcbffabe3ae5c9935bb21e3e94852R178, and I don't see the `server` variable being used anywhere.